content/docs/composition-vs-inheritance.md

```diff
@@ -78,15 +78,15 @@ function App() {
 }
 ```
 
-[**Try it on CodePen**](https://codepen.io/gaearon/pen/gwZOJp?editors=0010)
+[**Փորձել CodePen-ում**](https://codepen.io/gaearon/pen/gwZOJp?editors=0010)
 
-React elements like `<Contacts />` and `<Chat />` are just objects, so you can pass them as props like any other data. This approach may remind you of "slots" in other libraries but there are no limitations on what you can pass as props in React.
+React էլեմենտները, ինչպիսիք են `<Contacts />`-ը և `<Chat />`-ը ուղղակի օբյեկներ են. այսպիսով դուք կարող եք փոխանցել նրանց որպես props ուրիշ ցանկացած տվյալի նման։ Այս մոտեցումը կարող է ձեզ հիշեցնել ուրիշ գրադարաններում «slot»-երի գաղափարը, սակայն այստեղ չկան սահմանափակումներ, թե ինչ կփոխանցեք React-ին որպես props։
 
-## Specialization {#specialization}
+## Մասնագիտացում {#specialization}
 
-Sometimes we think about components as being "special cases" of other components. For example, we might say that a `WelcomeDialog` is a special case of `Dialog`.
+Հաճախ մենք կոմպոնենտների մասին մտածում ենք ինչպես ուրիշ կոմպոնենտների «հատուկ դեպք»։ Օրինակ, մենք կարող ենք ասել, որ `WelcomeDialog`-ը `Dialog`-ի հատուկ դեպք է։
 
-In React, this is also achieved by composition, where a more "specific" component renders a more "generic" one and configures it with props:
+React-ում, սա նույնպես ձեռք է բերվում կոմպոզիցիայով, որտեղ ավելի «սպեցիֆիկ» կոմպոնենտը արտապատկերում է ավելի «ընդհանուր» կոմպոնենտի և կարգավորում է այն prop-երով.
```

Comment:
Mars Exploration Program -> Մարսի հետազոտության ծրագիր
How should we refer to you? -> Ինչպե՞ս կարող ենք կապվել ձեզ հետ
Sign Me Up! -> Գրանցեք ինձ
Welcome aboard, ${this.state.login}! -> Բարի՛ գալուստ ${this.state.login}